purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -101,7 +101,8 @@ class Purchases internal constructor(
     private val identityManager: IdentityManager,
     private val subscriberAttributesManager: SubscriberAttributesManager,
     @set:JvmSynthetic @get:JvmSynthetic internal var appConfig: AppConfig,
-    private val customerInfoHelper: CustomerInfoHelper
+    private val customerInfoHelper: CustomerInfoHelper,
+    private val mainHandler: Handler = Handler(Looper.getMainLooper())
```

Comment:
By moving the initialization here, we ensure this is executed before the `init` so we don't have the issue where this can be null during the `init`